app/views/recommendation/index.html.erb

```diff
@@ -27,15 +27,45 @@
 
     <div class="container">
 
-<p class="subtitle">
-       The Smartest Reccomendation For You Would Be For You To Have The Honour Of Consuming The Following Products
-       <div class = "column"
+      <p class="subtitle">
+       <strong>The Smartest Reccomendation For You Would Be For You To Have The Honour Of Consuming The Following Products </strong>
+       <div class = "column">
        <%= render "products" %>
        </div>
         <br>
        The Smart Reccomendation Engine Has Faith In You & The Reccomended Products.<strong>Good Luck </strong>.
 
       </p>
+      <br>
+
+       <div class = "columns">
```

Comment:
Food for thought: maybe we could find a way to wrap up creating columns neatly into partials, to save some duplication between templates.